integrations/maltiverse.py

```diff
@@ -0,0 +1,599 @@
+            results.append(
+                maltiverse_alert(
+                    alert_id=alert["id"],
+                    ioc_dict=md5_ioc,
+                    ioc_name=md5,
+                )
+            )
+
+    if "syscheck" in alert and "sha1_after" in alert["syscheck"]:
+        debug("# Maltiverse: SHA1 checksum present in the alert")
+        sha1 = alert["syscheck"]["sha1_after"]
+
+        if sha1_ioc := maltiverse_api.sample_get_by_sha1(sha1):
+            results.append(
+                maltiverse_alert(
+                    alert_id=alert["id"],
+                    ioc_dict=sha1_ioc,
+                    ioc_name=sha1,
+                )
+            )
+
+    if "data" in alert and "srcip" in alert["data"]:
+        debug("# Maltiverse: Source IP Address present in the alert")
+        ipv4 = alert["data"]["srcip"]
+
+        if not ipaddress.IPv4Address(ipv4).is_private:
+            if ipv4_ioc := maltiverse_api.ip_get(ipv4):
+                results.append(
+                    maltiverse_alert(
+                        alert_id=alert["id"],
+                        ioc_dict=ipv4_ioc,
+                        ioc_name=ipv4,
+                    )
+                )
+
+    if "data" in alert and "hostname" in alert["data"]:
+        debug("# Maltiverse: Hostname present in the alert")
+        hostname = alert["data"]["hostname"]
+
+        if hostname_ioc := maltiverse_api.hostname_get(hostname):
+            results.append(
+                maltiverse_alert(
+                    alert_id=alert["id"],
+                    ioc_dict=hostname_ioc,
+                    ioc_name=hostname,
+                )
+            )
+
+    if "data" in alert and "url" in alert["data"]:
+        debug("# Maltiverse: URL present in the alert")
+        url = alert["data"]["url"]
+        urlchecksum = hashlib.sha256(url.encode("utf-8")).hexdigest()
+
+        if url_ioc := maltiverse_api.url_get(urlchecksum):
+            results.append(
+                maltiverse_alert(
+                    alert_id=alert["id"],
+                    ioc_dict=url_ioc,
+                    ioc_name=url,
+                    ioc_ref=urlchecksum,
+                )
+            )
+
+    return results
+
+
+def send_event(msg: str, agent: dict = None):
+    """Send an event to the Wazuh Manager.
+
+    Parameters
+    ----------
+    msg : str
+        The event message.
+    agent : dict, optional
+        The agent information.
+    """
+    if not agent or agent["id"] == "000":
+        string = f"1:maltiverse:{json.dumps(msg)}"
+    else:
+        location = "[{0}] ({1}) {2}".format(
+            agent["id"],
+            agent["name"],
+            agent["ip"] if "ip" in agent else "any",
+        )
+        location = location.replace("|", "||").replace(":", "|:")
+        string = f"1:{location}->maltiverse:{json.dumps(msg)}"
+
+    debug(string)
+    sock = socket(AF_UNIX, SOCK_DGRAM)
+    sock.connect(SOCKET_ADDR)
+    sock.send(string.encode())
```

Comment:
Is needed to validate that the size of the event doesn't exceed `65535` bytes. This is the max size that analysisd can process.

Something like
https://github.com/wazuh/wazuh/blob/d9d8cdd350b3d8ea20da905d4ae75828c10035b8/framework/wazuh/core/wazuh_queue.py#L219-L223

Reply:
I added exception handling for this case as in `aws-s3.py` `send_msg()` method